test/index/vars/vars.snapshot.cc

```diff
@@ -28,11 +28,13 @@
   }
   
   struct S {
+//       ^ definition [..] S#
     int x;
     static int y;
   };
   
   int f(S s) {
+//      ^ reference [..] S#
 //        ^ definition local 8
     return s.x + S::y;
```

Comment:
There is no reference for `S` here because we short-circuit and return nullopt in `saveVarDecl` for static and non-static members (to be added later), so there is no walk up the FQN.

Reply:
Hmm, not sure if this reasoning is still valid, I should double-check if there is no `TraverseNestedNameSpecifierLoc` call here.